packages/styles/scss/components/text-input/_text-input.scss

```
@@ -0,0 +1,384 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+//-----------------------------
+// Text
+//-----------------------------
+@use '../../theme' as *;
+@use '../../type' as *;
+@use '../../utilities/component-reset';
+@use '../../utilities/focus-outline' as *;
+@use '../../utilities/button-reset';
+@use '../form';
+
+/// Text input styles
+/// @access private
+/// @group text-input
+@mixin text-input {
+  .#{$prefix}--text-input {
+    @include reset;
+    @include type-style('body-short-01');
+    @include focus-outline('reset');
+
+    width: 100%;
+    height: rem(40px);
+    padding: 0 $carbon--spacing-05;
+    border: none;
+    border-bottom: 1px solid $border-strong;
+    background-color: $field;
+    color: $text-primary;
+    transition: background-color $duration--fast-01 motion(standard, productive),
+      outline $duration--fast-01 motion(standard, productive);
+
+    &:focus,
+    &:active {
+      @include focus-outline('outline');
+    }
+
+    &-wrapper svg[hidden] {
+      display: none;
+    }
+  }
+
+  // TODO V11: Remove xl selector
+  .#{$prefix}--text-input--xl,
+  .#{$prefix}--text-input--lg {
+    height: rem(48px);
+  }
+
+  .#{$prefix}--text-input--sm {
+    height: rem(32px);
+  }
+
+  .#{$prefix}--password-input {
+    padding-right: $carbon--spacing-08;
+  }
+
+  .#{$prefix}--text-input--sm.#{$prefix}--password-input {
+    padding-right: $carbon--spacing-07;
+  }
+
+  .#{$prefix}--text-input--lg.#{$prefix}--password-input {
+    padding-right: $carbon--spacing-09;
+  }
+
+  .#{$prefix}--text-input::placeholder {
+    @include placeholder-colors;
+  }
+
+  // V11: Possibly deprecate
+  .#{$prefix}--text-input--light {
+    background-color: $field-02;
+  }
+
+  //-----------------------------
+  // Disabled & Error icon spacing
+  //-----------------------------
+  .#{$prefix}--text-input__field-wrapper {
+    position: relative;
+    display: flex;
+    width: 100%;
+  }
+
+  .#{$prefix}--text-input__invalid-icon {
+    position: absolute;
+    // top/transform used to center invalid icon in IE11
+    top: 50%;
+    right: $carbon--spacing-05;
+    fill: $support-error;
+    transform: translateY(-50%);
+  }
+
+  .#{$prefix}--text-input__invalid-icon--warning {
+    fill: $support-warning;
+  }
+
+  .#{$prefix}--text-input__invalid-icon--warning path:first-of-type {
+    fill: $carbon__black-100;
```

I think this should be updated to be the following:
```suggestion
    fill: $black-100;
```